subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/locking/LockingInteractionsIntegrationTest.groovy

```diff
@@ -17,13 +17,15 @@
 package org.gradle.integtests.resolve.locking
 
 import org.gradle.integtests.fixtures.AbstractHttpDependencyResolutionTest
+import org.gradle.integtests.fixtures.resolve.ResolveTestFixture
 
 class LockingInteractionsIntegrationTest extends AbstractHttpDependencyResolutionTest {
 
     def lockfileFixture = new LockfileFixture(testDirectory: testDirectory)
 
     def setup() {
         settingsFile << "rootProject.name = 'locking-interactions'"
+        new ResolveTestFixture(buildFile, "lockedConf").addDefaultVariantDerivationStrategy()
```

Comment:
This uses internal APIs to add the derivation strategy. 

Is there any reason we can't make the test apply the jvm-ecosystem plugin instead?